src/docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion_TemperatureDependentVolumetricHeatCapacity/Example.rst

```diff
@@ -49,7 +49,7 @@ Results and benchmark
 A good agreement between the results obtained using GEOS and the analytical results is shown in the figure below:
 
 
-.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion/nonLinearThermalDiffusionAroundWellbore.py
+.. plot:: docs/sphinx/advancedExamples/validationStudies/wellboreProblems/nonLinearThermalDiffusion_TemperatureDependentVolumetricHeatCapacity/plot.py
```

Comment:
Why some difference between GEOS and FDM Non-linear are observed for this cooling case? If the case can be refined to show a better match?

Reply:
I have tried different experements but I can not get a ferfect fit. This might be because the reference result is also numercial, and there might be numercial errors from both sides.